templates/games/card-game.html

```diff
@@ -1,28 +1,159 @@
 {% extends "base.html" %}
 {% load static %}
 {% block extra_css %}
-<link rel="stylesheet" href="{% static 'card-game.css' %}">
+    <link rel="stylesheet" href="{% static '/css/card-game.css' %}">
 {% endblock %}
 
 {% block content %}
 <!-- Content Goes here -->
-<h1>card game page</h1>
```

Comment:
Shall we capitalise the sentence? IMO it will look better like this: Card Game Page

Reply:
That was removed and change with my project name